katsdpimager/preprocess.cpp

```diff
@@ -39,14 +39,16 @@ namespace py = pybind11;
 template<int P>
 struct vis_t
 {
+    /* NB: be careful about modifying the type or order of fields: for efficiency, memcmp is
```

Comment:
Nifty trick.

Reply:
I would have preferred it if the compiler could have worked it out for itself.